README.md

```diff
@@ -12,13 +12,13 @@ Script in Ruby to store Students and teachers
 
 1. Clone the repo
 
-   ```sh
+   ```sh 
    git@github.com:hunter4466/OOP_school_library.git
    ```
 
 ### Usage
 
-1. Execute program
+1. Execute program 
 
    ```sh
    ruby main.rb
```

Comment:
- [ ] Your README shows `Morse Code`. Kindly add the appropriate README content applicable to this project.